### ISSUE TITLE
Notify Users of Card Assignment

### ISSUE THREAD

**Issue by samuraii78** [mattermost/focalboard#2917]
When creating a new card in boards and assigning it to a user other than the creator of the card, shouldn't that user be automatically notified of the assignment? Is there an option to be set to have that?

**Comment by lopeici** [comment]
I agree. This is fundamental. not sure why this has not been picked up!

**Comment by xhildashazivari** [comment]
Hello, is there a recent update regarding this functionality?
